### PR TITLE
Gas update

### PR DESCRIPTION
**Pull request by aaronTufts**
Note: I failed to successfully squash the commit that had access keys in it (prior to me updating the code to be a template for devs to fill in). I've destroyed all exposed access keys.

### CODE REVIEW

**Comment by aaronTufts** [comment]
I failed to successfully squash the commit that had access keys in it (prior to me updating the code to be a template for devs to fill in). I've destroyed all exposed access keys.

File: gas-station/typescript/src/sponsored_examples.ts [hunk 1]
```diff
@@ -1,163 +1,240 @@
...
+//   KeyClient, 
+//   WalletClient, 
+//   createSuiClient, 
+//   buildGaslessTransactionBytes 
+// } from "@shinami/clients";
+
+// // 1. Copy your Node and Wallet Services Testnet Api Key Values
+// const NODE_TESTNET_ACCESS_KEY = "sui_testnet_bb0ae4d25b7e51a98c8fcb267babc1b6";
+// const WALLET_SERVICES_ONLY_KEY = "3ccf23b29a99cb4d4c2974e590575f23";
+// const ALL_SERVICES_TESTNET_KEY = "sui_testnet_a683fdb22331e5a9ef146b4dc2f5cf67";
```
Comment (jachen-sh) [inline gas-station/typescript/src/sponsored_examples.ts]: Remove these and destroy the keys as well

File: gas-station/typescript/src/sponsored_examples.ts [hunk 2]
```diff
@@ -1,163 +1,240 @@
...
+// const sessionToken = await keyClient.createSession(INVISIBLE_WAL_SECRET);
+// const senderSig = await walletClient.signTransactionBlock(
+//   INVISIBLE_WAL_ID,
+//   sessionToken,
+//   sponsoredResponse.txBytes
+// );
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
```
Comment (jachen-sh) [inline gas-station/typescript/src/sponsored_examples.ts]: Move this down to replace step 1 below

File: gas-station/typescript/src/sponsored_examples.ts [hunk 3]
```diff
@@ -1,163 +1,240 @@
...
+//     txb.moveCall({
+//       target: "0xfa0e78030bd16672174c2d6cc4cd5d1d1423d03c28a74909b2a148eda8bcca16::clock::access",
+//       arguments: [txb.object('0x6')]
+//     });
+//   }
+// });
+
+// // 4. Set the Id and Secret values for a previously created Invisible Wallet
+// const INVISIBLE_WAL_SECRET = "akeuy32wekd";
+// const INVISIBLE_WAL_ID = "aaron";
```
Comment (jachen-sh) [inline gas-station/typescript/src/sponsored_examples.ts]: replace these with the tutorial stubs e.g. `{{walletSecret}}`

File: gas-station/typescript/src/sponsored_examples.ts [hunk 4]
```diff
@@ -1,163 +1,240 @@
...
+//   GAS_BUDGET,
+//   { showEffects: true },
+//   "WaitForLocalExecution"
+// );
+
+// // 9. View the result
+// console.log(signSponsorExecuteResponse);
+
+
+// -- A different way to generate gasless transaction bytes as a base64 encoded string -- //
```
Comment (jachen-sh) [inline gas-station/typescript/src/sponsored_examples.ts]: Move this section to be aligned with its position in the tutorial

File: gas-station/typescript/src/sponsored_examples.ts [hunk 5]
```diff
@@ -1,163 +1,240 @@
...
+//   arguments: [txb.object('0x6')]
+// });
+//   // generate the bcs serialized transaction data without any gas object data
+//   const gaslessPayloadBytes = await txb.build({ client: nodeClient, onlyTransactionKind: true});
+
+//   // convert the byte array to a base64 encoded string
+//   const gaslessPayloadBase64 = btoa(
+//       gaslessPayloadBytes
+//           .reduce((data, byte) => data + String.fromCharCode(byte), '')
+//   );
```
Comment (jachen-sh) [inline gas-station/typescript/src/sponsored_examples.ts]: nit: newline ending

File: gas-station/typescript/src/sponsored_examples.ts [hunk 6]
```diff
@@ -1,163 +1,240 @@
...
-  console.log("Execution Status:", executeResponse.effects?.status.status);
-};
-
-sponsorTransactionE2E();
+console.log("Transaction Digest:", executeResponse.digest);
+console.log("Execution Status:", executeResponse.effects?.status.status);
+
+
+
+// -- Generate a KeyPair from a private key or passphrase -- //
```
Comment (jachen-sh) [inline gas-station/typescript/src/sponsored_examples.ts]: Move this section to be aligned with where it shows up in the tutorial

File: gas-station/typescript/src/sponsored_examples.ts (hunk 1 above)
Comment (aaronTufts) [reply gas-station/typescript/src/sponsored_examples.ts]: Done.

File: gas-station/typescript/src/sponsored_examples.ts (hunk 3 above)
Comment (aaronTufts) [reply gas-station/typescript/src/sponsored_examples.ts]: I ended up moving these to the Invisible Wallet tutorial so I didn't have the same code twice, and linking to that. I'll make that update in those examples

File: gas-station/typescript/src/sponsored_examples.ts (hunk 2 above)
Comment (aaronTufts) [reply gas-station/typescript/src/sponsored_examples.ts]: I ended up moving these to the Invisible Wallet tutorial so I didn't have the same code twice, and linking to that.

File: gas-station/typescript/src/sponsored_examples.ts (hunk 4 above)
Comment (aaronTufts) [reply gas-station/typescript/src/sponsored_examples.ts]: fixed

File: gas-station/typescript/src/sponsored_examples.ts (hunk 5 above)
Comment (aaronTufts) [reply gas-station/typescript/src/sponsored_examples.ts]: fixed

File: gas-station/typescript/src/sponsored_examples.ts (hunk 6 above)
Comment (aaronTufts) [reply gas-station/typescript/src/sponsored_examples.ts]: fixed

**Review by jachen-sh — APPROVED** [review]

**Pull request merged**